code/iaas/model/src/main/java/io/cattle/platform/core/cleanup/TableCleanup.java

```diff
@@ -169,6 +172,37 @@ private void cleanupServiceEventTable(Date cutoff) {
             log.info("[Rows Deleted] service_event={}", rowsDeleted);
         }
     }
+    
+    private void cleanupMountTable(Date cutoff) {
+    	List<Long> purgeInstanceIds = create()
+    			.select(INSTANCE.ID)
+    			.from(INSTANCE)
+    			.where(INSTANCE.STATE.eq("purged"))
+    			.fetch().into(Long.class);
+    	List<Long> localOrHostBindMountVolumeIds = create()
+    			.select(VOLUME.ID)
+    			.from(VOLUME)
+    			.where(VOLUME.DATA.like("%\"isHostPath\":true%"))
+    			.or(VOLUME.DATA.like("%\"driver\":\"local\"%"))
```

Comment:
The above two lines are scary bc they are such a potentially big scan. Need to think about it/consult with @ibuildthecloud or @vincent99